src/app/exhibitor/packages/page.tsx

```diff
@@ -18,6 +19,25 @@ export const metadata: Metadata = {
 		"The packages we offer for exhibitors at Armada. Choose between bronze, silver and gold."
 }
 
+function FAQItem({
+	children,
+	title
+}: {
+	children: React.ReactNode
+	title: string
+}) {
+	return (
+		<AccordionItem key={title} value={title} className="border-none">
```

Comment:
the `key` prop here doesn't really do anything since it is not in a list, hence it can be removed, if we want to apply this component to a list we set the key prop of the entire component outside this function instead of inside